src/brewtarget.cpp

```diff
@@ -388,10 +388,16 @@ const QDir Brewtarget::getConfigDir(bool *success)
    return dir.absolutePath() + "/";
 
 #elif defined(Q_OS_WIN) // Windows OS.
+   if (!qApp)
+   {  //QApplication instance doesn't exist yet
+      if (success)
+         *success = false;
+      return QString();
```

Comment:
It actually happens everytime because of the static member: 
Line 88: QDir Brewtarget::userDataDir = getConfigDir();
which is initialized before calling main(), i.e. before creating the QApplication.